api/v1/runtime_types.go

```diff
@@ -33,6 +35,21 @@ const (
 	AnnotationGardenerCloudDelConfirmation = "confirmation.gardener.cloud/deletion"
 )
 
+const (
+	LabelKymaInstanceID      = "kyma-project.io/instance-id"
+	LabelKymaRuntimeID       = "kyma-project.io/runtime-id"
+	LabelKymaShootName       = "kyma-project.io/shootName"
+	LabelKymaRegion          = "kyma-project.io/region"
+	LabelKymaName            = "kyma-project.io/kyma-name"
```

Comment:
KEB had label key `operator.kyma-project.io/kyma-name` (with `operator.`). Was it intended to be different in KIM?

Reply:
Fixed

Reply:
Also you can see https://github.com/kyma-project/infrastructure-manager/blob/main/docs/adr/001-provisioning.md